lib/auth/permissions.go

```diff
@@ -371,8 +374,10 @@ func GetCheckerForBuiltinRole(clusterName string, clusterConfig services.Cluster
 				role.String(),
 				services.RoleSpecV3{
 					Allow: services.RoleConditions{
-						Namespaces:    []string{services.Wildcard},
-						ClusterLabels: services.Labels{services.Wildcard: []string{services.Wildcard}},
+						Namespaces:     []string{services.Wildcard},
+						AppLabels:      services.Labels{services.Wildcard: []string{services.Wildcard}},
+						ClusterLabels:  services.Labels{services.Wildcard: []string{services.Wildcard}},
+						DatabaseLabels: services.Labels{services.Wildcard: []string{services.Wildcard}},
```

Comment:
What about KubernetesLabels?

Reply:
Originally I was only adding the permissions that were required to make the tests pass, but this would make sense I think with the new defaults. Done.